app.py

```diff
@@ -0,0 +1,174 @@
+import os
+from llama_stack_client import LlamaStackClient
+from tools import SANDBOX_DIR, TOOLS, run_tool
+import json
+
+MODEL_ID = "meta-llama/Llama-3.1-405B-Instruct-FP8"
+
+# Number of code review cycles
+CODE_REVIEW_CYCLES = 7
```

Comment:
With 7 there's a bunch of files that don't get completed.

So sometimes for some iterations it gets worse.

The tool calling implemented by llama stack/json parsing might not be that good